go/inst/instance_dao.go

```diff
@@ -3068,6 +3155,79 @@ func FigureInstanceKey(instanceKey *InstanceKey, thisInstanceKey *InstanceKey) (
 	return figuredKey, nil
 }
 
+// PopulateGroupReplicationInformation obtains information about Group Replication  for this host as well as other hosts
+// who are members of the same group (if any). This method swallows up any errors, interpreting them as if the host does
+// not support group replication (e.g. because the group replication plugin is not loaded, therefore the corresponding
+// performance schema tables and global variables are not known to the server.
+func PopulateGroupReplicationInformation(instance *Instance, db *sql.DB) {
+	q := `
+	SELECT
+		MEMBER_ID,
+		MEMBER_HOST,
+		MEMBER_PORT,
+		MEMBER_STATE, 
+		MEMBER_ROLE,
+		@@global.group_replication_group_name,
+		@@global.group_replication_single_primary_mode
+  	FROM
+		performance_schema.replication_group_members
```

Comment:
Does this require any special privileges? (I don't know)

Reply:
Confirmed, a user with only `SELECT` privilege in `performance_schema.replication_group_members` can run the query.

Reply:
So, it _does_ require special privileges. Can you please document this in https://github.com/openark/orchestrator/blob/master/docs/configuration-discovery-basic.md ?

Reply:
Done